core/src/main/java/tech/pegasys/eth2signer/core/multikey/MultiKeyArtifactSignerProvider.java

```diff
@@ -30,23 +30,23 @@
 
   private static final Logger LOG = LogManager.getLogger();
 
-  private final SigningMetadataTomlConfigLoader signingMetadataTomlConfigLoader;
+  private final SigningMetadataConfigLoader signingMetadataConfigLoader;
```

Comment:
nit: can we genericise this further - eg configLoader?

Reply:
It's still specific to signing metadata config files so don't think it can be